config/routes.rb

```diff
@@ -6,7 +6,9 @@
   resource :user_sessions, only: [:create]
   root 'home#index'
   get  "first-edition" => "home#first_edition_projects"
-  get "sponsors" => "home#sponsors"
+  get  "sponsors" => "home#sponsors"
+  get  "mentee-profile" => "mentee_profile#index"
```

Comment:
how about we use `recource :mentee_profile, only: [:index]` here as we'll be later adding edit for mentee